docs/how-to/store-and-retrieve.md

```diff
@@ -0,0 +1,43 @@
+# How to store and retrieve device settings
+
+Ophyd-async has the functionality to easily store the current values of all of a [](#Device)'s [](#SignalRW)s in a fully customisable format, and to load that state to the Device during a [Bluesky plan](https://nsls-ii.github.io/bluesky/plans.html). For a plan which requires a Device to be in, or close to a known state, we can, before running an experiment plan:
```

Comment:
We have intersphinx on this, so can reference objects in remote repos. The tricky thing is finding the right link. You can use `myst-inv` to do this:
```bash
$ myst-inv -n *plan* https://blueskyproject.io/bluesky/main/objects.inv
```
In this case I think the correct reference is `plans`, so this would be:
```suggestion
Ophyd-async has the functionality to easily store the current values of all of a [](#Device)'s [](#SignalRW)s in a fully customisable format, and to load that state to the Device during a [Bluesky plan](inv:bluesky#plans). For a plan which requires a Device to be in, or close to a known state, we can, before running an experiment plan:
```